edgelet/contrib/config/linux/config.yaml

```diff
@@ -16,11 +16,33 @@
 # Configures the identity provisioning mode of the daemon.
 #
 # Supported modes:
-#     manual   - using an iothub connection string
+#     manual   - using an iothub connection string or a X.509 identity certificate
```

Comment:
`a X.509` -> `an X.509`

Reply:
There are still five of these, some of which are also missing the `.` in `X.509`

Reply:
Done. Thanks for keeping me honest.